core/src/test/scala/unit/kafka/server/GroupCoordinatorBaseRequestTest.scala

```diff
@@ -421,26 +483,31 @@ class GroupCoordinatorBaseRequestTest(cluster: ClusterInstance) {
   protected def leaveGroupWithOldProtocol(
     groupId: String,
     memberIds: List[String],
+    groupInstanceId: String = null,
     expectedLeaveGroupError: Errors,
-    expectedMemberErrors: List[Errors]
+    expectedMemberErrors: List[Errors],
+    version: Short
   ): Unit = {
-    if (memberIds.size != expectedMemberErrors.size) {
-      fail("genericGroupLeave: memberIds and expectedMemberErrors have unmatched sizes.")
-    }
-
     val leaveGroupRequest = new LeaveGroupRequest.Builder(
       groupId,
-      memberIds.map(memberId => new MemberIdentity().setMemberId(memberId)).asJava
-    ).build()
+      memberIds.map(memberId => new MemberIdentity()
+        .setMemberId(memberId)
+        .setGroupInstanceId(groupInstanceId)
```

Comment:
This is weird because the group instance id is supposed to be unique for each member.